### PR TITLE
Pgcraft cleanups

### PR DESCRIPTION
**Pull request by mbouron**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
   `NGL_CAP_BLOCK`, `NGL_CAP_INSTANCED_DRAW`, `NGL_CAP_SHADER_TEXTURE_LOD`,
   `NGL_CAP_TEXTURE_2D_ARRAY`, `NGL_CAP_TEXTURE_3D`, `NGL_CAP_TEXTURE_CUBE`,
   `NGL_CAP_UINT_UNIFORMS`
+- GLSL ngl_tex*() declarations (users must now use texture() and textureLod()
+  directly)
```
Comment (ubitux) [inline CHANGELOG.md:51]: please mention that `ngl_texvideo()` is still there and the recommended abstract picking method

Comment (mbouron) [reply CHANGELOG.md:51]: Done.

**Review by ubitux — APPROVED** [review]
LGTM; feel free to add a few backticks to the programming keywords in the `CHANGELOG.md` entry

**Comment by mbouron** [comment]
> LGTM; feel free to add a few backticks to the programming keywords in the `CHANGELOG.md` entry

Done.

**Pull request merged**